src/layers/arc-layer/arc-layer.js

```diff
@@ -25,25 +25,35 @@ const glslify = require('glslify');
 const RED = [255, 0, 0];
 const BLUE = [0, 0, 255];
 
+const defaultGetPosition0 = x => x.position0;
+const defaultGetPosition1 = x => x.position1;
```

Comment:
since we are doing the refactoring now, do you think it is more intuitive to name these two: sourcePosition and targetPosition, instead of position0 and position1, as we only have two vertices for the arc and the line layer?